### DISCUSSION TITLE
Safelist col-span-{x} classes is not working.

### DISCUSSION THREAD

**Discussion by cgtarmenta** [tailwindlabs/tailwindcss#9653] [Help] [answered]
**What version of Tailwind CSS are you using?**
Tailwind: 3.2.1

**What build tool (or framework if it abstracts the build tool) are you using?**
Vue: 3.2.41
Vite:  3.1.8
PostCSS: 8.4.18

**What version of Node.js are you using?**
Node: 16.17.1

**What browser are you using?**
Browser: Chrome, Edge

**What operating system are you using?**

OS: Windows 11 pro / WSL2 

**Reproduction URL**
[Reproduction](https://play.tailwindcss.com/DUZthw58vc)

**Describe your issue**
Safelist col-span-{x} classes is not working, nor in dev(running just vite) or production build.
I've tried all methods(list the 12 individual classes, various patterns) to safelist them, but the only thing that worked was to create a var in the code like:

```
const classes = [
  'col-span-1',
  'col-span-2',
  'col-span-3',
  'col-span-4',
  'col-span-5',
  'col-span-6',
  'col-span-7',
  'col-span-8',
  'col-span-9',
  'col-span-10',
  'col-span-11',
  'col-span-12',
]
```
That worked even if the code above is commented, but that's the only way I found to make the classes available

**Comment by adamwathan** [comment, marked as answer]
Hey! Make sure to lowercase the “l”, the option is `safelist` not `safeList`.

**Reply by cgtarmenta** [reply]
Man I hate myself for this issue, I feel so dumb right now XD

now I got other issue, less important:

```
safelist: [
    {
      pattern: /.*/,
      variants: ['hover', 'focus', 'dark'],
    },
  ],
```

When I made that, I'm getting this:

```
<--- Last few GCs --->

[207287:0x5d0e100]    40556 ms: Mark-sweep (reduce) 1988.8 (2082.1) -> 1987.9 (2082.3) MB, 860.2 / 0.0 ms  (average mu = 0.386, current mu = 0.002) allocation failure; scavenge might not succeed
[207287:0x5d0e100]    41408 ms: Mark-sweep (reduce) 1989.1 (2082.3) -> 1988.5 (2082.8) MB, 850.4 / 0.0 ms  (average mu = 0.230, current mu = 0.002) allocation failure; scavenge might not succeed


<--- JS stacktrace --->

```

Since I'm dealing with a really complex json, that was made to work with bootstrap, I need to make all the CSS programmatically, and I'm trying to avoid safelisting class by class is there a way to get all the classes at least in dev?
